MANIFEST.in

```diff
@@ -1,5 +1,3 @@
 include MANIFEST.in
 include LICENSE
 include run_scapy
-include scapy/VERSION
-recursive-exclude test *
```

Comment:
Looks like the warnings are gone but the test directory seems to end up in the sdist somehow even though `pyproject.toml` explicitly excludes it:
```sh
$ tar tzvf dist/scapy-2.5.0.dev72.tar.gz '*/test'
drwxr-xr-x root/root         0 2023-04-09 13:07 scapy-2.5.0.dev72/test/
-rw-rw-r-- root/root      5126 2023-04-08 20:52 scapy-2.5.0.dev72/test/testsocket.py
```
I would have expected it to somehow end up in the wheel too but it isn't there:
```sh
$ unzip -l dist/scapy-2.5.0.dev72-py3-none-any.whl | grep 'test/'
```

Reply:
Let's ignore the warnings then and add it back.

Reply:
Makes sense. I suspect it's a setuptools issue but I have to admit I don't know what it was exactly.